# Q3
A bolted assembly of two parts is shown in Figure Q3, is to support an external load, P
= 80 kN. The steel bolt is fine thread and the modulus of elasticity is 200 GPa and reused
connection. The joint stiffness factor for the design is 0.286. The parts are made of cast
iron with a modulus of elasticity of 70 GPa. Assume the factor of safety for the joint is 2. Determine;

i) The stiffness constant for the part of the effective area of the part is 2500 mm^2

ii) The stiffness constant for the bolt and size of the bolt, d.

iii) The suitable metric specification class number for the bolt and force on the
bolt.

In [9]:
# Required
from math import pi, e, pow, log

In [10]:
# All units in m
PLoad = 80E+3
print('Fine thread. Class type is not given')
Eb = 200E+9
print('Reusable connection')
C = 0.286
Ep = 70E+9
FOS = 2
Nbolt = 1
# From diagram
L = 0.05

Fine thread. Class type is not given
Reusable connection


i) The stiffness constant for the part of the effective area of the part is 2500 mm^2

Find kp if Ap is given.

$$kp=\frac{A_pE_p}{L}$$

Convert $2500mm^2$ to $m^2$

$$2500mm \times \frac{1m}{1000mm} \times \frac{1m}{1000mm}=2500\times 10^{-6} m$$

Substitute into kp equation

$$kp=\frac{(2500\times10^{-6})(70\times10^{9})}{50\times10^{-3}}$$

In [11]:
Ap=2500E-6
kp=Ap*Ep/L
print(f'kp={kp:.3e}')

kp=3.500e+09


ii) The stiffness constant for the bolt and size of the bolt, d.

The joint stiffness factor.

$$C=\frac{k_b}{k_b+k_p}$$

$$0.286=\frac{k_b}{k_b+3.5\times10^{9}}$$

In [13]:
kb=C*kp/(1-C)
print(f'kb={kb:.3e}')

kb=1.402e+09


Bolt stiffness constant

$$k_b=\frac{A_bE_b}{L}$$

$$1.402\times10^{9}=\frac{A_b(200\times10^{9})}{50\times10^{-3}}$$

where $A_b$ is cross-sectional area of the bolt

$$A_b=\frac{\pi d^2}{4}$$

$$1.402\times10^{9}=\frac{(\frac{\pi d^2}{4})(200\times10^{9})}{50\times10^{-3}}$$

In [19]:
d_bolt=(kb*L*4/(pi*Eb))**0.5
print(f'Minimum bolt diameter={d_bolt:.4f}m = {d_bolt*1000:.2f}mm')
print(f'Rounding up to nearest standard bolt size,d=22mm')

Minimum bolt diameter=0.0211m = 21.12mm
Rounding up to nearest standard bolt size,d=22mm


iii) The suitable metric specification class number for the bolt and force on the
bolt.

Total force on bolt
$$F_b = CP+F_i$$

If factor of safety and number of bolt is considered.

Total force on bolt
$$F_b = \frac{CPn}{N_{bolt}}+F_i$$

Since Proof strength is unknown, we cannot calculate Fb. Here assume $F_b=F_{proof}=S_p A_t$

$$S_p A_t = \frac{CPn}{N_{bolt}}+F_i$$

No of bolt is 1. Factor of Safety,n=2.

Reusable joint, $F_i=0.75S_p A_t$

$$S_p A_t = \frac{CP(2)}{1}+0.75S_p A_t$$

Refer Table 2.1. These is no bolt size 22mm. The nearest is 24 mm.

For fine thread, 24mm, $A_t=384mm^2=384\times10^{-6}$

C=0.286

$$S_p(384\times10^{-6})= \frac{(0.286)(80\times10^3)(2)}{1}+0.75 S_p(384\times10^{-6})$$

In [ ]:
At=384E-6
Sp=C*PLoad*FOS/(At-(0.75*At))
print(f'Proof strength={Sp:.2f}Pa')
print(f'Proof strength={Sp/1E6:.2f}MPa')

Proof strength=476666666.67Pa
Proof strength=476.67Pa


Minimum proof strength for the design is 476.67MPa.
Therefore from Table 2.2, the next higher Proof strength is 600MPa.

Class number 8.8